pkg/client/client.go

```diff
@@ -125,8 +127,21 @@ func formatNodeAddress(w io.Writer, elem *models.NodeAddressingElement, title, p
 	return false
 }
 
+func timeSince(since time.Time) string {
+	out := "never"
+	if !since.IsZero() {
+		// Poor man's implementtion of time.Truncate(). Can be refined
+		// when we rebase to go 1.9
+		t := time.Since(since)
+		t -= t % time.Second
+		out = t.String() + " ago"
+	}
+
+	return out
+}
+
 // FormatStatusResponse writes a StatusResponse as a string to the writer
-func FormatStatusResponse(w io.Writer, sr *models.StatusResponse) {
+func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, allControllers bool) {
```

Comment:
Could you docuement what `allControllers` does?

Reply:
fixed